changelogs/fragments/ovirt_host_network_check_for_empty_user_opts

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ovirt_host_network: check for empty user_opts (https://github.com/ansible/ansible/pull/47283).
```

Comment:
@abadger
abadger just now Member

The problem in the changelog is the ": ". In yaml, that has a special meaning, the separator between a dictionary's key and value elements.

The easiest fix is to change the ": " into a " - ".